financial-connections/src/main/java/com/stripe/android/financialconnections/ui/components/TopAppBar.kt

```diff
@@ -20,25 +21,30 @@ import androidx.compose.ui.unit.dp
 import com.stripe.android.financialconnections.R
 import com.stripe.android.financialconnections.ui.FinancialConnectionsPreview
 import com.stripe.android.financialconnections.ui.LocalNavHostController
+import com.stripe.android.financialconnections.ui.LocalReducedBranding
 import com.stripe.android.financialconnections.ui.theme.FinancialConnectionsTheme
 
 @Composable
 internal fun FinancialConnectionsTopAppBar(
-    title: @Composable () -> Unit = {
-        Icon(
-            painter = painterResource(id = R.drawable.stripe_logo),
-            contentDescription = null // decorative element
-        )
-    },
     elevation: Dp = 0.dp,
     showBack: Boolean = true,
     onCloseClick: () -> Unit
 ) {
     val localBackPressed = LocalOnBackPressedDispatcherOwner.current
         ?.onBackPressedDispatcher
     val navController = LocalNavHostController.current
+    val reducedBranding = LocalReducedBranding.current
     TopAppBar(
-        title = title,
+        title = if (reducedBranding) {
+            { Text("") }
```

Comment:
I'm _guessing_ this is to keep the same sizing or something. Would it be better to use a `Box` for this usecase?

Reply:
Actually not needed. Added an empty composable function instead - https://github.com/stripe/stripe-android/pull/5949/commits/180ea2d7a122ab85c488380d4593afe2a5b6bc6f